p3/dataaccess.py

```diff
@@ -5,10 +5,20 @@
 from assopy import models as amodels
 from assopy import utils as autils
 from p3 import models
+from django.conf import settings
 from django.contrib.auth.models import User
 from django.contrib.contenttypes.models import ContentType
 
-cache_me = cachef.CacheFunction(prefix='p3:')
```

Comment:
I'm wondering if it would be better to change the CacheFunction in conference.cachef, what do you think?

Reply:
I was thinking about it, but didn't actually try if it would work. I could make a little test and push another patch if it works.